translate/storage/factory.py

```diff
@@ -63,18 +63,18 @@
 def _examine_txt(storefile):
     """Determine the true filetype for a .txt file"""
     if isinstance(storefile, six.string_types) and os.path.exists(storefile):
-        storefile = open(storefile)
+        storefile = open(storefile, 'rb')
     try:
         start = storefile.read(600).strip()
     except AttributeError:
         raise ValueError("Need to read object to determine type")
     # Some encoding magic for Wordfast
     from translate.storage import wordfast
-    if wordfast.TAB_UTF16 in start.split("\n")[0]:
+    if wordfast.TAB_UTF16 in start.split(b"\n")[0]:
         encoding = 'utf-16'
     else:
         encoding = 'iso-8859-1'
-    start = start.decode(encoding).encode('utf-8')
+    start = start.decode(encoding)
```

Comment:
These cleanups seem unrelated to `open`. If they are please put them in a separate commit.